db/db.go

```diff
@@ -126,8 +126,5 @@ func (m *memKVStore) Commit(b KVStoreBatch) (e error) {
 
 // NewOnDiskDB instantiates an on-disk KV store
 func NewOnDiskDB(cfg config.DB) KVStore {
```

Comment:
There's no need to have `NewOnDiskDB`. People should instantiate BoltDB explicitly.

Reply:
NewOnDiskDB function is now used in other files, so I think it is little bit tricky to change it whole stuff